ksqldb-engine/src/main/java/io/confluent/ksql/engine/EngineExecutor.java

```diff
@@ -321,39 +319,27 @@ PullQueryResult executeTablePullQuery(
         ));
       }
 
-      final String stmtLower = statement.getMaskedStatementText().toLowerCase(Locale.ROOT);
-      final String messageLower = e.getMessage().toLowerCase(Locale.ROOT);
-      final String stackLower = Throwables.getStackTraceAsString(e).toLowerCase(Locale.ROOT);
-
-      // do not include the statement text in the default logs as it may contain sensitive
-      // information - the exception which is returned to the user below will contain
-      // the contents of the query
-      if (messageLower.contains(stmtLower) || stackLower.contains(stmtLower)) {
-        final StackTraceElement loc = Iterables
-            .getLast(Throwables.getCausalChain(e))
-            .getStackTrace()[0];
-        LOG.error("Failure to execute pull query {} {}, not logging the error message since it "
-            + "contains the query string, which may contain sensitive information. If you "
-            + "see this LOG message, please submit a GitHub ticket and we will scrub "
-            + "the statement text from the error at {}",
-            routingOptions.debugString(),
-            queryPlannerOptions.debugString(),
-            loc);
-      } else {
-        LOG.error("Failure to execute pull query. {} {}",
-            routingOptions.debugString(),
-            queryPlannerOptions.debugString(),
-            e);
-      }
-      LOG.debug("Failed pull query text {}, {}", statement.getMaskedStatementText(), e);
-
-      throw new KsqlStatementException(
-          e.getMessage() == null
-              ? "Server Error" + Arrays.toString(e.getStackTrace())
-              : e.getMessage(),
+      QueryLogger.error(
```

Comment:
Since we have now audited all parsing and statement exceptions to remove query text, we no longer need to search through the stacktrace before we log it.

This wasn't an effective mechanism before anyway, since it was inconsistently applied and because it only matches when the exact query appears in the stacktrace as opposed to a fragment of the query.

It's better to just use the QueryLogger universally.